cerberus/errors.py

```diff
@@ -70,7 +70,7 @@
 # TODO remove KEYSCHEMA AND VALUESCHEMA with next major release
 KEYSRULES = KEYSCHEMA = ErrorDefinition(0x83, 'keysrules')
 VALUESRULES = VALUESCHEMA = ErrorDefinition(0x84, 'valuesrules')
-BAD_ITEMS = ErrorDefinition(0x8f, 'items')
+BAD_ITEMS = ErrorDefinition(0x8F, 'items')
```

Comment:
that is an unnecessary change.

Reply:
this violates black reformatting rule in travis pipeline.